app/src/pages/Courses.tsx

```diff
@@ -80,36 +124,122 @@ const Courses = () => {
           {data.length ? (
             <>
               {/* Header and create course button */}
-              <div className="flex flex-row no-wrap">
+              <div className="flex flex-row no-wrap items-center">
                 <h1 className="text-3xl font-bold flex-1">
                   Confira seus cursos
                 </h1>
                 <div className="flex flex-row gap-5">
                   <label
                     htmlFor="course-create"
                     onClick={CourseManager}
-                    className="std-button flex modal-button space-x-2 cursor-pointer"
+                    className="std-button flex modal-button space-x-2 cursor-pointer px-12 py-2.5"
                   >
-                    <PencilSquareIcon className="w-8 h-8" />
-                    <p className="font-normal ">Criar novo curso</p>{" "}
+                    <p className="font-normal flex items-center "><span className="text-3xl mr-2">+</span> Novo Curso</p>{" "}
                     {/** Create new course */}
                   </label>
                   {/* Course guide button shows a tutorial*/}
-                  <CourseGuideButton />
+                  {/* <CourseGuideButton /> */}
                 </div>
               </div>
               {/* Card/compact view toggle and filters */}
-              <div className="h-10 my-8 bg-grayLight">
+              <div className="my-8 flex justify-between">
                 {/* TODO: Implement card/compact view toggle */}
+                <div className="flex ">
+
+                <button  onClick={() => handleGridOrList('grid')}>
+                  <div id="GridView" className={`p-2 rounded-l-lg border border-grayLight ${isGridView ? "bg-primary": ''}`}>
+                      <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M8 12C9.1 12 10 12.9 10 14C10 15.1 9.1 16 8 16C6.9 16 6 15.1 6 14C6 12.9 6.9 12 8 12ZM8 6C9.1 6 10 6.9 10 8C10 9.1 9.1 10 8 10C6.9 10 6 9.1 6 8C6 6.9 6.9 6 8 6ZM8 0C9.1 0 10 0.9 10 2C10 3.1 9.1 4 8 4C6.9 4 6 3.1 6 2C6 0.9 6.9 0 8 0ZM2 12C3.1 12 4 12.9 4 14C4 15.1 3.1 16 2 16C0.9 16 0 15.1 0 14C0 12.9 0.9 12 2 12ZM2 6C3.1 6 4 6.9 4 8C4 9.1 3.1 10 2 10C0.9 10 0 9.1 0 8C0 6.9 0.9 6 2 6ZM2 0C3.1 0 4 0.9 4 2C4 3.1 3.1 4 2 4C0.9 4 0 3.1 0 2C0 0.9 0.9 0 2 0ZM14 12C15.1 12 16 12.9 16 14C16 15.1 15.1 16 14 16C12.9 16 12 15.1 12 14C12 12.9 12.9 12 14 12ZM14 6C15.1 6 16 6.9 16 8C16 9.1 15.1 10 14 10C12.9 10 12 9.1 12 8C12 6.9 12.9 6 14 6ZM14 0C15.1 0 16 0.9 16 2C16 3.1 15.1 4 14 4C12.9 4 12 3.1 12 2C12 0.9 12.9 0 14 0Z" fill={isGridView ? "#FFF" : "#A1ACB2"}/>
```

Comment:
Maybe it is just because I am a n00b, but I have no clue what is going on here 😵 Maybe a comment with a short explanation would be helpful for future students?